contributing_to_docs/doc_guidelines.adoc

```diff
@@ -743,21 +743,86 @@ Replace all public IP addresses with an address from the following blocks. These
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
+====
+
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+Using a lead-in link sentence that adheres to guidelines maintains consistency across OpenShift Container Platform documentation.
+
+==== Lead-in link sentence style for non-xref situations
+
+For situations where you cannot use the `xref` attribute, such as referencing a module within a module, ensure that you adhere to the following guidance for lead-in link sentences.
+
+* Use the following format when pointing to a section in the same document:
+
+[source,text,subs="+quotes"]
+----
+For more information, see "<section_name>".
+----
+
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+
+==== Lead-in link sentence style for the xref attribute
+
+For situations where you can use the `xref` attribute, adhere to the following guidance:
+
+* When pointing to a section (module) that exists in the same document (assembly), create an _Additional resources_ section in the assembly file and then place the `xref` as a bulleted item under the _Additional resources_ section. For example:
+
+[source,text]
+----
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc#installation-config-parameters-vsphere[Installation configuration parameters]
+----
+
+* When pointing to a link in a section (module) that exists in another OpenShift Container Platform document (assembly), use the following format:
+
+[source,text]
+----
+For more information on <topic>, see xref:<path_to_file>["<section_name>"] in _<document_name>_.
+----
+
+Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
+
+For more information about using `xref`, see xref:links-conditionals-assemblies[Links, hyperlinks, and cross references].
+
+==== Lead-in link sentence style for the link attribute
+
+For situations where you can use the `link` attribute, adhere to the following guidance:
+
+* When pointing to a link that exists external to Red Hat's customer portal documentation, use the `link` attribute, place the link in an _Additional resources_ section, and use the `<document_heading_name> (<document_source>)` format. For example:
```

Comment:
To avoid conflict with [add'l resources rules](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)
```suggestion
* When pointing to a link that exists external to Red Hat's customer portal documentation, use the `link` attribute, and use the `<document_heading_name> (<document_source>)` format. For example:
```

Reply:
Thanks. Updated.

Reply:
Added the "why" to this entry:

![Uploading Screenshot from 2024-04-15 15-52-31.png…]()